backend/worker.py

```
@@ -0,0 +1,53 @@
+    channel.queue_declare(queue='glaucoma_queue')
+    
+    print(" [*] Connected! Listening on 'glaucoma_queue'...")
+    print("=====================================================")
+
+    # auto_ack=False ensures messages aren't lost if the worker crashes
+    channel.basic_consume(queue='glaucoma_queue', on_message_callback=callback, auto_ack=False)
+    channel.start_consuming()
+    
+except Exception as e:
+    import traceback
+    print(f"\n[-] Queue connection error occurred.")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)